spec/system/task_spec.rb

```diff
@@ -85,28 +79,25 @@
 
       it '既にステータスが完了のタスクのステータスを変更した場合、Taskの完了日が更新されないこと' do
         # TODO: FactoryBotのtraitを利用してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id, status: :done, completion_date: Time.current.yesterday)
-        visit edit_project_task_path(project, task)
+        visit edit_project_task_path(project, completed_task)
         select 'todo', from: 'Status'
         click_button 'Update Task'
         expect(page).to have_content('todo')
         expect(page).not_to have_content(Time.current.strftime('%Y-%m-%d'))
-        expect(current_path).to eq project_task_path(project, task)
+        expect(current_path).to eq project_task_path(project, completed_task)
       end
     end
   end
 
   describe 'Task削除' do
+    let!(:task) { create(:task, project_id: project.id) }
```

Comment:
ここもassociationを定義するとproject_idの指定をしなくて済みますね。
```suggestion
    let!(:task) { create(:task) }
```